Tasks/PackerBuildV1/src/taskParameters.ts

```diff
@@ -47,6 +48,7 @@ export default class TaskParameters {
                 this.customTemplateLocation = tl.getPathInput(constants.CustomTemplateLocationInputType, true, true);
                 console.log(tl.loc("ParsingCustomTemplateParameters"));
                 this.customTemplateParameters = JSON.parse(tl.getInput("customTemplateParameters"));
+                this.packerVersionString = tl.getInput(constants.PackerVersionInputName);
```

Comment:
.trim() the whitespaces in input

Reply:
I assume getInput wont fail if the input is not specified

Reply:
tl.getInput trims the whitespaces itself, also getInput throws an error when the required parameter to it given.